src/DB_Replace.php

```diff
@@ -2,9 +2,140 @@
 
 namespace JustCoded\WP\Composer;
 
-// TODO: create db replace command
-
+use Composer\Script\Event;
+use JustCoded\WP\Composer\Helpers\Array_Helper;
+use JustCoded\WP\Composer\Helpers\Process_Controller;
+use JustCoded\WP\Composer\Helpers\Scripts_Helper;
+use PHP_CodeSniffer\Tokenizers\PHP;
 
+/**
+ * Replace srting in DB
+ *
+ * Usage:
+ *      wp:dbUpdate -- --search='old string' --replace='new string' --method --tables
+ *
+ * Options:
+ *      --search    Old string
+ *      --replace   New string
+ *      --method    Method replace only content columns/string data columns or replace all data columns full/simple (default simple)
+ *      --tables    List of tables separated by comma (default all)
+ *
+ */
 class DB_Replace {
 
+	public static function db_replace( Event $event ) {
+		$args    = Scripts_Helper::parse_arguments( $event->getArguments() );
+		$options = self::checkOptions( $args );
+
+		if ( ! $options || empty( $options['search'] ) ) {
+			self::printError();
+		}
+
+		$replace_param = self::prepareOptions( $options );
+		self::initReplace( $replace_param );
+	}
+
+	/**
+	 * Print error MSG
+	 *
+	 * @throws \ReflectionException
+	 */
+	protected static function printError() {
+		echo PHP_EOL;
+		$rc = new \ReflectionClass( get_class() );
+		print_r( '========================================================' );
+		echo PHP_EOL;
+		print_r( '|| Something wrong. Read documentation and try again. ||' );
+		echo PHP_EOL;
+		print_r( '========================================================' );
+		echo PHP_EOL;
+		print_r( $rc->getDocComment() );
+		echo PHP_EOL;
+		die;
+	}
+
+	/**
+	 * @param array $replace_param
+	 */
+	protected static function initReplace( $replace_param = array() ) {
```

Comment:
method name is incorrect. this method does not "init", actually it "replace" or update database, but not init.

"init" keyword is used when you set some variables or properties, which are required for other code parts.